### PR TITLE
docs(docs-infra): for accessibility and responsive,  iframe should be…

### PR DESCRIPTION
**Pull request by MeAkib**
For accessibility and responsive, iframe should be in video container, contain some space horizontally, and title attribute.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

https://angular.io/guide/typed-forms
image

https://angular.io/guide/standalone-components
image

Issue Number: N/A

## What is the new behavior?
image
image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/src/styles/1-layouts/doc-viewer/_doc-viewer.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
   position: relative;
   padding-bottom: 56.25%; /* 16:9 */
   height: 0;
+  margin: 20px 0;
```
Comment (MarkTechson) [inline aio/src/styles/1-layouts/doc-viewer/_doc-viewer.scss:10]: Can you let me know why you added an extra margin to the styles?

Comment (akib1997) [reply aio/src/styles/1-layouts/doc-viewer/_doc-viewer.scss:10]: The `.video-container` class does not have its own space. So after `h2` tag almost sticking. Should I update?

image

**Comment by akib1997** [comment]
> Deployed aio for [56ae191](https://github.com/angular/angular/commit/56ae19121285d76351951e06d51f9683a1eb6bdc) to: https://ng-dev-previews-fw--pr-angular-angular-51475-bcusm8n5.web.app
> 
> _**Note**: As new commits are pushed to this pull request, this link is updated after the preview is rebuilt._
Current output
image

File: aio/src/styles/1-layouts/doc-viewer/_doc-viewer.scss (hunk 1 above)
Comment (akib1997) [reply aio/src/styles/1-layouts/doc-viewer/_doc-viewer.scss:10]: https://github.com/angular/angular/pull/51475#issuecomment-1693132893

Comment (akib1997) [reply aio/src/styles/1-layouts/doc-viewer/_doc-viewer.scss:10]: https://github.com/angular/angular/pull/51475#issuecomment-1691100697

image

**Review by MarkTechson — APPROVED** [review]
LGTM! Thanks for this contribution.

**Review by AndrewKushnir — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 3c28506f8e5afff33a2086cdc6cfbe11a6a7b35f.

**Pull request closed without merging**